packages/components/src/item-group/styles.ts

```diff
@@ -8,32 +8,35 @@ import { css } from '@emotion/react';
  */
 import { CONFIG, COLORS } from '../utils';
 
-export const unstyledButton = css`
-	appearance: none;
-	border: 1px solid transparent;
-	cursor: pointer;
-	background: none;
-	text-align: start;
-
-	svg,
-	path {
-		fill: currentColor;
-	}
-
-	&:hover {
-		color: ${ COLORS.ui.theme };
-	}
-
-	&:focus-visible {
-		box-shadow: 0 0 0 var( --wp-admin-border-width-focus )
-			var(
-				--wp-components-color-accent,
-				var( --wp-admin-theme-color, ${ COLORS.ui.theme } )
-			);
-		// Windows high contrast mode.
-		outline: 2px solid transparent;
-	}
-`;
+export const unstyledButton = ( as: 'a' | 'button' ) => {
```

Comment:
Should we also add an explicit font-size to normalize the sizes between button and anchor? ([Like this](https://github.com/WordPress/gutenberg/blob/af38c95e3777b562bdf509647c228c78d68c467a/packages/components/src/dropdown-menu-v2/styles.ts#L162))

<img width="168" alt="Different font sizes in Storybook" src="https://github.com/WordPress/gutenberg/assets/555336/7385fe96-5950-4c66-a4b5-40cff9142592">

In Storybook we can see the font sizes will be different due to the user agent styles.

Reply:
Please see: https://github.com/WordPress/gutenberg/pull/52495#discussion_r1261271103